pkg/server/testserver.go

```diff
@@ -296,10 +297,17 @@ func (ts *TestServer) Start(params base.TestServerArgs) error {
 }
 
 // ExpectedInitialRangeCount returns the expected number of ranges that should
-// be on the server after initial (asynchronous) splits have been completed,
-// assuming no additional information is added outside of the normal bootstrap
-// process.
+// be on the server after initial (asynchronous) splits have been completed and
+// system migrations have been run, assuming no additional information is added
+// outside of the normal bootstrap process.
 func ExpectedInitialRangeCount() int {
+	// 1 has to be added for the eventlog table that gets created by a migration.
```

Comment:
It does only get run in tests, but a lot of tests use the default `(*server).Start` method, and thus get migrations run unless they're explicitly turned off. And it does seem desirable for the test startup to be as similar to normal startup as possible, as annoying as it might be to fix up all the places that creating a table during startup breaks.